masterbase/lib.py

```diff
@@ -600,20 +600,20 @@ def add_loser(engine: Engine, steam_id: str) -> None:
         conn.commit()
 
 
-def add_report(engine: Engine, session_id: str, target_steam_id: str | None) -> None:
+def add_report(engine: Engine, session_id: str, target_steam_id: str | None, reason: str) -> None:
     """Submit a hackusation to the database."""
```

Comment:
wait why is target_steam_id nullable here? (not touched in this MR but idk why it is)

Reply:
if we can make this not null lets do it in this PR